test/support/utils.rb

```diff
@@ -73,18 +74,32 @@ def create_pseudo_terminal(boot_options: "-r debug/run")
               when %r{Really quit\? \[Y/n\]}
                 cmd = @queue.pop
                 write.puts(cmd)
+              when /REPL ERROR/
```

Comment:
How about to check it in `@last_backlog`?

Reply:
updated 👍